carbonmark-api/package.json

```diff
@@ -7,7 +7,7 @@
     "build": "rm -rf ./dist && tsc",
     "dev": "tsnd --respawn --transpile-only --exit-child src/server.ts --port 3003",
     "test": "jest --coverage",
-    "lint": "eslint . --ext .ts",
+    "lint": "eslint . --ext .ts && tsc --noEmit && prettier --check --loglevel warn .",
```

Comment:
also redundant prettier check see other comment

Reply:
and Done